common-theurgy.lic

```diff
@@ -142,12 +142,30 @@ module DRCTH
     DRCI.put_away_item?(DRC.left_hand, container)
   end
 
+  def sprinkle_holy_water?(theurgy_supply_container, water_holder, target)
+    did_sprinkle_holy_water = (
```

Comment:
Capturing the result and returning it isn't necessary unless you feel the var name adds expressiveness

The "Ruby way" would be
```suggestion
def sprinkle_holy_water?(theurgy_supply_container, water_holder, target)
      DRCI.get_item?(water_holder, theurgy_supply_container) && sprinkle?(water_holder, target) && DRCI.put_away_item?(water_holder, theurgy_supply_container)
end
```

(or add some newlines if you prefer that)

Ruby functions implicitly return the last thing in the function

Reply:
You and @asechrest said the same thing :) Great minds and all that. Let me swap that. I originally thought the variable name made things more explicit, but if the name of the function isn't explicit enough... that's a different problem.

Reply:
Like this?